src/code42cli/cmds/legal_hold.py

```diff
@@ -124,6 +148,30 @@ def show(state, matter_id, include_inactive=False, include_policy=False):
         echo("")
 
 
+@legal_hold.command()
+@matter_id_option(False, "Filter results by legal hold UID.")
+@click.option(
+    "--event-type",
+    type=click.Choice(LEGAL_HOLD_EVENT_TYPES),
+    help="Filter results by event types.",
+)
+@click.option("--begin", **BEGIN_DATE_DICT)
+@click.option("--end", **END_DATE_DICT)
+@format_option
+@sdk_options()
+def search_events(state, matter_id, event_type, begin, end, format):
+    """Report on legal hold events."""
```

Comment:
@annie-payseur This is the command's help text: `Report on legal hold events.`

Reply:
I suggest changing to `Tools for getting legal hold event data.` (which is more consistent with the help text for `security-data`)